package.json

```diff
@@ -25,6 +26,8 @@
     "github-cache": "^1.1.0",
     "hydrolysis": "^1.19.3",
     "nodegit": "^0.5.0",
+    "nomnom": "^1.8.1",
```

Comment:
Recently I saw someone mention that `nomnom` is no longer maintained, and `yargs` is a preferred alternative.